libs/platform/ebpf_epoch.c

```diff
@@ -640,6 +645,10 @@ _ebpf_epoch_get_release_epoch(_Out_ int64_t* release_epoch)
             lowest_epoch = min(lowest_epoch, _ebpf_epoch_cpu_table[cpu_id].epoch_state.epoch);
         }
 
+        // Compute the cutoff time for inactive threads.
+        uint64_t inactive_thread_expiry_time =
```

Comment:
nit: maybe a misnomer, since isn't the time the thread expires.  maybe "max_last_used_time"?
```suggestion
        uint64_t max_last_used_time =
```

Reply:
This time represents the lower for last used time for thread entries that are inactive, but not stale.

Inactive threads in the time interval [0, stale_thread_cutoff) are stale and should be removed.
Inactive threads in the time interval [stale_thread_cutoff, now] are inactive but not stale.